src/typescript/_namespaces/ts.ts

```diff
@@ -4,5 +4,6 @@ export * from "../../compiler/_namespaces/ts";
 export * from "../../jsTyping/_namespaces/ts";
 export * from "../../services/_namespaces/ts";
 export * from "../../server/_namespaces/ts";
+export * from "../../executeCommandLine/_namespaces/ts";

```

Comment:
Also if this is going to be in public API, we could just move this to compiler instead of in separate folder ?

Reply:
Yeah, it's just one file, so that would work too.